Makefile

```diff
@@ -1026,6 +1026,12 @@ compare: out/mkcmp out/minikube
 generate-licenses:
 	./hack/generate_licenses.sh
 
+.PHONY: gomodtidy
+gomodtidy: ## run go mod tidy everywhere needed
```

Comment:
tidy?

Reply:
tidy could mean many other tidies could be node, site ,... I think it auto-completes very nicely and intuitively
you just type make gom..  and it auto completes itself, dont have to remember it was called Tidy